### ISSUE TITLE
Example fails on Ubuntu 15.04

### ISSUE THREAD

**Issue by RPiAwesomeness** [WatchBeam/beam-client-python#2]
I was going to attempt to create my own Beam bot, just as a side project for my own amusement.

So, I `git clone`-d the repository and then run `python3 setup.py build && sudo python3 setup.py install`. That all went off without a hitch.

However, when I went to run the example code (after updating my `config.py`) I got this:

```
$ python3 example.py 
Connecting to wss://chat2-dal07.beam.pro:443/ws
ERROR:tornado.general:Uncaught exception, closing connection.
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/tornado-4.2-py3.4-linux-i686.egg/tornado/iostream.py", line 504, in _handle_events
    self._handle_connect()
  File "/usr/local/lib/python3.4/dist-packages/tornado-4.2-py3.4-linux-i686.egg/tornado/iostream.py", line 1359, in _handle_connect
    do_handshake_on_connect=False)
  File "/usr/local/lib/python3.4/dist-packages/tornado-4.2-py3.4-linux-i686.egg/tornado/netutil.py", line 510, in ssl_wrap_socket
    return context.wrap_socket(socket, **kwargs)
  File "/usr/lib/python3.4/ssl.py", line 365, in wrap_socket
    _context=self)
  File "/usr/lib/python3.4/ssl.py", line 542, in __init__
    raise ValueError("check_hostname requires server_hostname")
ValueError: check_hostname requires server_hostname
ERROR:tornado.application:Exception in callback None
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/tornado-4.2-py3.4-linux-i686.egg/tornado/ioloop.py", line 866, in start
    handler_func(fd_obj, events)
  File "/usr/local/lib/python3.4/dist-packages/tornado-4.2-py3.4-linux-i686.egg/tornado/stack_context.py", line 275, in null_wrapper
    return fn(*args, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/tornado-4.2-py3.4-linux-i686.egg/tornado/iostream.py", line 504, in _handle_events
    self._handle_connect()
  File "/usr/local/lib/python3.4/dist-packages/tornado-4.2-py3.4-linux-i686.egg/tornado/iostream.py", line 1359, in _handle_connect
    do_handshake_on_connect=False)
  File "/usr/local/lib/python3.4/dist-packages/tornado-4.2-py3.4-linux-i686.egg/tornado/netutil.py", line 510, in ssl_wrap_socket
    return context.wrap_socket(socket, **kwargs)
  File "/usr/lib/python3.4/ssl.py", line 365, in wrap_socket
    _context=self)
  File "/usr/lib/python3.4/ssl.py", line 542, in __init__
    raise ValueError("check_hostname requires server_hostname")
ValueError: check_hostname requires server_hostname
Socket closed [Code 1005] Connection refused
Reestablishing the socket in 1 second.
^CTraceback (most recent call last):
  File "example.py", line 30, in <module>
    ioloop.IOLoop.instance().start()
  File "/usr/local/lib/python3.4/dist-packages/tornado-4.2-py3.4-linux-i686.egg/tornado/ioloop.py", line 841, in start
    event_pairs = self._impl.poll(poll_timeout)
KeyboardInterrupt
```

What's causing this? I'm on Ubuntu MATE 15.04, running Python 3.4.3. All of the required packages are fully up to date.

**Comment by dudeinamotorhome** [comment]
Looks like the hostname is not properly being passed from the config file to the function that is using the info. Did you copy config.example.py to config.py and edit config.py to suite your needs for login and channel id. I think the older version was config.example.json to config.json. Depending on the version you have.

**Comment by dudeinamotorhome** [comment]
Looks like the hostname isn't being read from the config file.  There might be a small typo somewhere, causing the info not to be read. I downloaded and tested on my raspberry pi earlier today and it worked fine for me. Are you using channel id # or channel name for channel in the config? If you are using the name, on linux type curl https://beam.pro/api/v1/channels/<name> replacing <name> with the streamer/channel name on the cl and the id should be listed at the very top under '{"id":###" ' you can also just post the link in a browser without the curl to get it as well. Could be causing the call to not format properly for the server. Just some thoughts, and you may have looked at them already, but thought I would reply just in case. Also check out https://gitter.im and join the MC Pro Hosting Beam Dev chat for more help, if you haven't already. A lot of fun and helpful people there as well.

**Comment by RPiAwesomeness** [comment]
@dudeinamotorhome I'm using the channel that I got via [https://beam.pro/api/v1/channels/ParadigmShift3d](https://beam.pro/api/v1/channels/ParadigmShift3d) and used the number from the id:"" section in the JSON that's returned.

From the traceback it looks like there's a bug in the bot example code that isn't interfacing with the python networking modules correctly.

I did pop in the MC Pro Hosting Beam Dev chat on gitter and they all agreed that the example code is entirely broken. Ben Baptist was there too (dev of BenBot) and he said the same thing. So, I just gave up on that & wrote my own code using manual websockets.
